src/App.vue

```diff
@@ -4,12 +4,26 @@
     :class="theme"
     :style="style"
     :dir="dir"
+    v-if="headerPosition === 'top'"
```

Comment:
instead of duplicating the code here, can we simply add a class and use css to invert the order?